### DISCUSSION TITLE
Error publishing to npm

### DISCUSSION THREAD

**Discussion by ScuffedItalian** [community/community#38043] [Actions]
I'll make this sweet and simple, I ran the command npm version major -m "v2" and then ran npm publish, the first time it published it I got no errors second time I added a lib required for package to work and reran the npm version major and the npm publish and got an error and it was with the lib I added, the file I added was the log. if you need any more resources to help solve this problem i'll give them to you.
[2022-11-02T04_20_39_351Z-debug-0.log](https://github.com/community/community/files/9915930/2022-11-02T04_20_39_351Z-debug-0.log)

**Comment by david-engelmann** [comment]
You'll probably get better responses if you submit this to a more appropriate place. This area is more for general github questions. [This link](https://www.npmjs.com/) might be a great resource, as well as the [npm github page](https://github.com/npm), the best section to submit this will probably be in the [cli repository issues in the npm github page](https://github.com/npm/cli/issues)
